index.js

```diff
@@ -0,0 +1,8 @@
+const divide = (a, b) => a / b;
+divide();
```

Comment:
These function calls aren't needed to pass the tests. Going forward, can you please omit them? Thanks!